examples/fscloud/version.tf

```diff
@@ -0,0 +1,10 @@
+terraform {
+  required_version = ">= 1.3.0"
+  required_providers {
+    # Use latest version of provider in non-basic examples to verify latest version works with module
+    ibm = {
+      source  = "IBM-Cloud/ibm"
+      version = ">= 1.54.0"
```

Comment:
this now need to be updated to >= 1.56.1 since https://github.com/terraform-ibm-modules/terraform-ibm-icd-rabbitmq/pull/22 was merged